plugins/modules/network_settings_intent.py

```diff
@@ -1750,26 +1763,27 @@ def update_reserve_pool(self, config):
         name = config.get("reserve_pool_details").get("name")
         result_reserve_pool = self.result.get("response")[1].get("reservePool")
         result_reserve_pool.get("response").update({name: {}})
-        self.log("Reserve Pool Cisco DNA Center Details " +
-                 str(self.have.get("reservePool").get("details")))
-        self.log("Reserve Pool User Details " +
-                 str(self.want.get("wantReserve")))
+        self.log("Reserve Pool Cisco DNA Center Details: {0}"
+                 .format(self.have.get("reservePool").get("details")), "DEBUG")
+        self.log("Reserve Pool User Details: {0}"
+                 .format(self.want.get("wantReserve")), "DEBUG")
 
         # Check pool exist, if not create and return
-        self.log(str(self.want.get("wantReserve").get("ipv4GlobalPool")))
+        self.log("ipv4_global_pool: {0}"
+                 .format(self.want.get("wantReserve").get("ipv4GlobalPool")), "DEBUG")
         site_name = config.get("reserve_pool_details").get("site_name")
         reserve_params = self.want.get("wantReserve")
         site_id = self.get_site_id(site_name)
         reserve_params.update({"site_id": site_id})
         if not self.have.get("reservePool").get("exists"):
-            self.log(str(reserve_params))
+            self.log("Reserve Pool Playbook Details: {0}".format(reserve_params), "DEBUG")
             response = self.dnac._exec(
                 family="network_settings",
                 function="reserve_ip_subpool",
                 params=reserve_params,
             )
             self.check_execution_response_status(response).check_return_status()
-            self.log("Ip Subpool Reservation Created Successfully")
+            self.log("Ip Subpool Reservation Created Successfully", "INFO")
```

Comment:
Successfully created IP Subpool reservation .. can we add name for this ? Or any identification